.github/ISSUE_TEMPLATE/Other.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Other
+about: Something else
+
+---
+
+<!-- Bug reports and Feature requests must use other templates, or will be closed -->
+<!-- Please ask questions on StackOverflow or the webpack Gitter (https://gitter.im/webpack/webpack). -->
```

Comment:
We should remove this line since it isn't specific to Open Horizon.

Reply:
Done @clementkng